package.json

```diff
@@ -47,7 +46,8 @@
       "uuid": "latest",
       "typestub-multibase": "latest",
       "typestub-multihashes": "latest",
-      "cron-parser": "latest"
+      "cron-parser": "latest",
+      "globby": "latest"
```

Comment:
is it really a good idea to have "latest" for the package version? Seems like this could cause problems if there is a release with a breaking change.

Reply:
Good point. I'll change versioning.